src/contracts/misc/GhoSteward.sol

```diff
@@ -0,0 +1,171 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import {IPoolAddressesProvider} from '@aave/core-v3/contracts/interfaces/IPoolAddressesProvider.sol';
+import {IPoolConfigurator} from '@aave/core-v3/contracts/interfaces/IPoolConfigurator.sol';
+import {IPool} from '@aave/core-v3/contracts/interfaces/IPool.sol';
+import {DataTypes} from '@aave/core-v3/contracts/protocol/libraries/types/DataTypes.sol';
+import {PercentageMath} from '@aave/core-v3/contracts/protocol/libraries/math/PercentageMath.sol';
+import {GhoInterestRateStrategy} from '../facilitators/aave/interestStrategy/GhoInterestRateStrategy.sol';
+import {IGhoToken} from '../gho/interfaces/IGhoToken.sol';
+import {IGhoSteward} from './interfaces/IGhoSteward.sol';
+
+/**
+ * @title GhoSteward
+ * @author Aave
+ * @notice Helper contract for managing risk parameters of the GHO reserve within the Aave Facilitator
+ * @dev This contract must be granted `PoolAdmin` in the Aave V3 Ethereum Pool and `BucketManager` in GHO Token
+ * @dev Only the Risk Council is able to action contract's functions.
+ * @dev Only the Aave DAO is able to extend the steward's lifespan.
+ */
+contract GhoSteward is IGhoSteward {
+  using PercentageMath for uint256;
+
+  /// @inheritdoc IGhoSteward
+  address public constant AAVE_SHORT_EXECUTOR = 0xEE56e2B3D491590B5b31738cC34d5232F378a8D5;
```

Comment:
You can also use aave-address-book

Reply:
Also, I would have used `Ownable` because with hardcoded permissions is way difficult to index then systems like the ShortExecutor to find them

Reply:
moved to `Ownable` to facilitate indexing